net/mwan3/files/lib/mwan3/common.sh

```diff
@@ -89,16 +89,12 @@ mwan3_get_mwan3track_status()
 	config_list_foreach "$1" track_ip mwan3_list_track_ips
 
 	if [ -n "$track_ips" ]; then
-		pid="$(pgrep -f "mwan3track $1$")"
-		if [ -n "$pid" ]; then
-			if [ "$(cat /proc/"$(pgrep -P $pid)"/cmdline)" = "sleep${MAX_SLEEP}" ]; then
-				tracking="paused"
```

Comment:
I'd like to keep the "paused" status. The tracker enters the "paused" state when the tracker is running but the interface is down, and it is helpful to know when it is in this state.

Reply:
I am referring to this line [here](https://github.com/openwrt/packages/blob/master/net/mwan3/files/usr/sbin/mwan3rtmon#L107). Should we not then also update the tables when a  tracking is `paused`?

Reply:
Tracking should only be paused when the interface is “down” so it will have no routing table, so no need to update.